phf/tests/test.rs

```diff
@@ -112,7 +112,7 @@ mod map {
         static MAP: phf::Map<&'static str, int> = phf_map!(
             "a" => 0,
         );
-        assert_eq!(Some(&0), MAP.get("a".to_string()[]));
```

Comment:
Parallel changes were made upstream and it was required to get it to compile. I dunno if it’s temporary.